docs/development/_index.md

```diff
@@ -44,16 +44,161 @@ and in a different terminal instance:
 make run-frontend
 ```
 
-## Build a Docker image
 
-The following command builds a Docker image for Headlamp from the current
+## Build the app
+
+You can build the app for Linux, Windows or Mac.
+
+Do so on the platform you are building for. That is build the mac app on a Mac, 
+and the linux app on a linux box.
+
+First we need to 
+```bash
+make backend frontend
+```
+
+Then choose the relevant command.
+
+```bash
+make app-linux
+```
+
+```bash
+make app-windows
+```
+
+```bash
+make app-mac
+```
+
+See the generated app files in app/dist/
+
+
+
```

Comment:
Is it common to add 3 newlines? Usually we add two but I am not sure what's standard.

Reply:
Updated the file to 2 lines before a header. Thanks.